text/2018-10-25-tikv-client-rust.md

```diff
@@ -0,0 +1,190 @@
+# Summary
+
+Introduce a full featured, official TiKV (and PD) Rust client. It will be intended to be used as a reference implementation, or to provide C-compatible binding for future clients.
```

Comment:
```suggestion
Introduce a full featured, official TiKV (and PD) Rust client. It is intended to be used as a reference implementation, or to provide C-compatible binding for future clients.
```

Reply:
Fixed. Please take another look of the newly added API usage section, thanks.